tests/cephadm/test_collocate_rgw_node_count.py

```diff
@@ -0,0 +1,35 @@
+from ceph.ceph import CommandFailed
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class Collocate_RGW_Node_Count(Exception):
```

Comment:
Change class name to `CollocateRGWNodeCountError`

Reply:
Thanks @vamahaja
Updated